lnrpc/rpc.proto

```diff
@@ -908,6 +918,85 @@ message SendToRouteRequest {
     Route route = 4;
 }
 
+message ChannelAcceptRequest {
```

Comment:
I don't think we need to include _all_ of this information. The keys on our side are auto generated, and don't provide any additional context. Minimally, we can leave off all the keying and revocation information.

Reply:
Done